src/themes/admin_default/html/layout_default.html.twig

```diff
@@ -214,7 +214,9 @@
 
             <footer class="footer footer-transparent d-print-none">
                 <div class="container-xl">
-                    <div>Copyright &copy; {{ now|format_date(pattern='yyyy') }}. All rights reserved. Powered by <a href="https://fossbilling.org" title="Billing system" target="_blank" rel="noopener">FOSSBilling {{ FOSSBillingVersion }}</a></div>
+                    <p>&copy; {{ now|format_date(pattern='yyyy') }}<a href="https://github.com/FOSSBilling">FOSSBilling contributors</a>.
```

Comment:
Looks like you're missing a space. Also you accidentally hardcoded the FOSSBilling version.
![image](https://github.com/user-attachments/assets/33685f9b-92a3-48af-b266-e5ebb106d3a6)

Reply:
Oops, fixed!